aiohttp/pytest_plugin.py

```diff
@@ -230,6 +230,23 @@ def proactor_loop():  # type: ignore
         yield _loop
 
 
+@pytest.fixture
+def selector_loop():  # type: ignore
+    if not PY_37:
+        policy = asyncio.get_event_loop_policy()
```

Comment:
Won't `3.8` also hit this branch?

Reply:
Nope. https://github.com/aio-libs/aiohttp/blob/master/aiohttp/helpers.py#L57

Reply:
Oh, so the var name is unfortunate.

Reply:
Yes, at first I got confused too...

Reply:
so maybe `... import PY_38 as AT_LEAST_PY_38` then?

Reply:
I think this is out of scope and the general idea of PY_3* should be reworked.